src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2216,11 +2216,11 @@ public enum ElementState
     public enum LacingStrategy
     {
         Disabled,
-        Auto,
```

Comment:
The addition of the auto in this location caused the silent failure in client code consuming the API as the enum values are constants and backed into the the client's assembly IL.

See: http://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net (Thanks @mjkkirschner )